doc/docs.md

```diff
@@ -1708,6 +1709,47 @@ fn main(){
 
 # Advanced Topics
 
+## Memory safety
```

Comment:
I'd call this subsection `unsafe`

Reply:
I changed it to `Memory-unsafe code`, hope that's OK.